platform/web-security/src/main/java/org/motechproject/security/constants/EmailConstants.java

```diff
@@ -0,0 +1,46 @@
+package org.motechproject.security.constants;
+
+/**
+ * Utility class for storing constants related to e-mails.
+ */
+public final class EmailConstants {
+
+    //Templates
+    public static final String RESET_MAIL_TEMPLATE = "resetMail.vm";
+    public static final String ONE_TIME_TOKEN_TEMPLATE = "oneTimeTokenMail.vm";
+    public static final String PASSWORD_CHANGE_REMINDER_TEMPLATE = "passwordChangeReminder.vm";
+    public static final String LOGIN_INFORMATION_TEMPLATE = "loginInfo.vm";
+    
+    //E-mail parameters
+    public static final String EMAIL_PARAM_FROM_ADDRESS = "fromAddress";
+    public static final String EMAIL_PARAM_TO_ADDRESS = "toAddress";
+    public static final String EMAIL_PARAM_MESSAGE = "message";
+    public static final String EMAIL_PARAM_SUBJECT = "subject";
+    
+    //Template parameters
+    public static final String TEMPLATE_PARAM_LINK = "link";
+    public static final String TEMPLATE_PARAM_USERNAME = "user";
+    public static final String TEMPLATE_PARAM_MESSAGES = "messages";
+    public static final String TEMPLATE_PARAM_LOCALE = "locale";
+    public static final String TEMPLATE_PARAM_EXPIRATION_DATE = "expirationDate";
+    public static final String TEMPLATE_PARAM_LAST_PASSWORD_CHANGE = "lastPasswordChange";
+    public static final String TEMPLATE_PARAM_DAYS_TILL_EXPIRE = "daysTillExpire";
+    public static final String TEMPLATE_PARAM_SERVER_URL = "serverUrl";
+    public static final String TEMPLATE_PARAM_EXTERNAL_ID = "externalId";
+
+    //E-mail subjects
+    public static final String RECOVERY_SUBJECT = "Motech Password Recovery";
```

Comment:
use the following notation - org.motechproject.security.email.passwordRecovery . This allows usage of wildcards in listeners.